portal/src/main/webapp/js/src/oncoprint/setup-oncoprint-improved.js

```diff
@@ -7,6 +7,44 @@ window.setUpOncoprint = function(ctr_id, config) {
 	var sample_to_patient = config.sample_to_patient;
 	var gene_data = config.gene_data;
 	
+        var genepanelValues = config.genepanelValues;
```

Comment:
didn't we decide this shouldn't be in the oncoprint setup? we should be loading this in the same place as we load the data, i.e. in studyview.js (which we should rename :sweat_smile: ) and just add the 'na' attribute to the data where necessary